src/App.vue

```diff
@@ -252,12 +252,11 @@
       open (packet) {
         this.id_tab = 0
 
-        if (this.id_open === packet) {
-          return this.id_open = null
+        if (this.open_ids.includes(packet)){
+          const index = this.open_ids.indexOf(packet);
+          return this.open_ids.splice(index, 1);
         }
-
-        window.packet = packet
```

Comment:
That was for quick debugging in console. I'd rather not resign from this behavior. Having an ability to manipulate over opened packet (packets in case of this PR) in console is a must have in my case.

Reply:
Ah, so that's what it was for. I have no problem with it so I will put it back. How can you use this?